### ISSUE TITLE
Rec model train on sentences. Why does it only recognize the first word in a sentence?

### ISSUE THREAD

**Issue by soufieneghribi** [PaddlePaddle/PaddleOCR#4847]
I'm training a model that recognizes sentences. I have 500.000 senteneces.
This is the configuration of my model.

The model only predicts the **first word of a sentence.** 

Does any one knows the reason. I have **96% evaluation** score and **86% train  accuracy**

here is an example:

![output139](https://user-images.githubusercontent.com/26394060/144850465-06f3d250-a6f5-48d4-ba0e-0d932b2c7b18.png)

result: بنت    0.98515606

Config: 
[2021/12/06 12:55:23] root INFO: Architecture :
[2021/12/06 12:55:23] root INFO:     Backbone :
[2021/12/06 12:55:23] root INFO:         model_name : small
[2021/12/06 12:55:23] root INFO:         name : MobileNetV3
[2021/12/06 12:55:23] root INFO:         scale : 0.5
[2021/12/06 12:55:23] root INFO:         small_stride : [1, 2, 2, 2]
[2021/12/06 12:55:23] root INFO:     Head :
[2021/12/06 12:55:23] root INFO:         fc_decay : 1e-05
[2021/12/06 12:55:23] root INFO:         name : CTCHead
[2021/12/06 12:55:23] root INFO:     Neck :
[2021/12/06 12:55:23] root INFO:         encoder_type : rnn
[2021/12/06 12:55:23] root INFO:         hidden_size : 48
[2021/12/06 12:55:23] root INFO:         name : SequenceEncoder
[2021/12/06 12:55:23] root INFO:     Transform : None
[2021/12/06 12:55:23] root INFO:     algorithm : CRNN
[2021/12/06 12:55:23] root INFO:     model_type : rec
[2021/12/06 12:55:23] root INFO: Eval :
[2021/12/06 12:55:23] root INFO:     dataset :
[2021/12/06 12:55:23] root INFO:         data_dir : /elya1/ocr_sentences
[2021/12/06 12:55:23] root INFO:         label_file_list : ['/elya1/trdg/sent_arabic_val.txt']
[2021/12/06 12:55:23] root INFO:         name : SimpleDataSet
[2021/12/06 12:55:23] root INFO:         transforms :
[2021/12/06 12:55:23] root INFO:             DecodeImage :
[2021/12/06 12:55:23] root INFO:                 channel_first : False
[2021/12/06 12:55:23] root INFO:                 img_mode : BGR
[2021/12/06 12:55:23] root INFO:             CTCLabelEncode : None
[2021/12/06 12:55:23] root INFO:             RecResizeImg :
[2021/12/06 12:55:23] root INFO:                 image_shape : [3, 32, 320]
[2021/12/06 12:55:23] root INFO:             KeepKeys :
[2021/12/06 12:55:23] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/12/06 12:55:23] root INFO:     loader :
[2021/12/06 12:55:23] root INFO:         batch_size_per_card : 256
[2021/12/06 12:55:23] root INFO:         drop_last : False
[2021/12/06 12:55:23] root INFO:         num_workers : 8
[2021/12/06 12:55:23] root INFO:         shuffle : False
[2021/12/06 12:55:23] root INFO: Global :
[2021/12/06 12:55:23] root INFO:     cal_metric_during_train : True
[2021/12/06 12:55:23] root INFO:     character_dict_path : ppocr/utils/dict/arabic_dict.txt
[2021/12/06 12:55:23] root INFO:     character_type : ch
[2021/12/06 12:55:23] root INFO:     checkpoints : None
[2021/12/06 12:55:23] root INFO:     debug : False
[2021/12/06 12:55:23] root INFO:     distributed : False
[2021/12/06 12:55:23] root INFO:     epoch_num : 500
[2021/12/06 12:55:23] root INFO:     eval_batch_step : [0, 2000]
[2021/12/06 12:55:23] root INFO:     infer_img : /home/sghribi/PaddleOCR/test/output139.png
[2021/12/06 12:55:23] root INFO:     infer_mode : False
[2021/12/06 12:55:23] root INFO:     log_smooth_window : 20
[2021/12/06 12:55:23] root INFO:     max_text_length : 140
[2021/12/06 12:55:23] root INFO:     pretrained_model : /home/sghribi/PaddleOCR/output/rec_arabic_lite_sentence/best_accuracy
[2021/12/06 12:55:23] root INFO:     print_batch_step : 10
[2021/12/06 12:55:23] root INFO:     save_epoch_step : 3
[2021/12/06 12:55:23] root INFO:     save_inference_dir : /home/sghribi/PaddleOCR/output/rec_arabic_lite_sentence
[2021/12/06 12:55:23] root INFO:     save_model_dir : /home/sghribi/PaddleOCR/output/rec_arabic_lite_sentence
[2021/12/06 12:55:23] root INFO:     use_gpu : True
[2021/12/06 12:55:23] root INFO:     use_space_char : False
[2021/12/06 12:55:23] root INFO:     use_visualdl : False
[2021/12/06 12:55:23] root INFO: Loss :
[2021/12/06 12:55:23] root INFO:     name : CTCLoss
[2021/12/06 12:55:23] root INFO: Metric :
[2021/12/06 12:55:23] root INFO:     main_indicator : acc
[2021/12/06 12:55:23] root INFO:     name : RecMetric
[2021/12/06 12:55:23] root INFO: Optimizer :
[2021/12/06 12:55:23] root INFO:     beta1 : 0.9
[2021/12/06 12:55:23] root INFO:     beta2 : 0.999
[2021/12/06 12:55:23] root INFO:     lr :
[2021/12/06 12:55:23] root INFO:         learning_rate : 0.001
[2021/12/06 12:55:23] root INFO:         name : Cosine
[2021/12/06 12:55:23] root INFO:     name : Adam
[2021/12/06 12:55:23] root INFO:     regularizer :
[2021/12/06 12:55:23] root INFO:         factor : 1e-05
[2021/12/06 12:55:23] root INFO:         name : L2
[2021/12/06 12:55:23] root INFO: PostProcess :
[2021/12/06 12:55:23] root INFO:     name : CTCLabelDecode
[2021/12/06 12:55:23] root INFO: Train :
[2021/12/06 12:55:23] root INFO:     dataset :
[2021/12/06 12:55:23] root INFO:         data_dir : /elya1/ocr_sentences
[2021/12/06 12:55:23] root INFO:         label_file_list : ['/elya1/trdg/sent_arabic_train.txt']
[2021/12/06 12:55:23] root INFO:         name : SimpleDataSet
[2021/12/06 12:55:23] root INFO:         transforms :
[2021/12/06 12:55:23] root INFO:             DecodeImage :
[2021/12/06 12:55:23] root INFO:                 channel_first : False
[2021/12/06 12:55:23] root INFO:                 img_mode : BGR
[2021/12/06 12:55:23] root INFO:             RecAug : None
[2021/12/06 12:55:23] root INFO:             CTCLabelEncode : None
[2021/12/06 12:55:23] root INFO:             RecResizeImg :
[2021/12/06 12:55:23] root INFO:                 image_shape : [3, 32, 320]
[2021/12/06 12:55:23] root INFO:             KeepKeys :
[2021/12/06 12:55:23] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/12/06 12:55:23] root INFO:     loader :
[2021/12/06 12:55:23] root INFO:         batch_size_per_card : 256
[2021/12/06 12:55:23] root INFO:         drop_last : True
[2021/12/06 12:55:23] root INFO:         num_workers : 8
[2021/12/06 12:55:23] root INFO:         shuffle : True

**Comment by soufieneghribi** [comment]
anyone?

**Comment by asif-ca** [comment]
@soufieneghribi can you share the steps you followed to fine-tune the arabic model?
